docs/UserGuide.adoc

```diff
@@ -207,41 +207,64 @@ exp/23-Nov-2020 a/21`
 
 ==== Add with system prompting
 
-In this mode, the user can add the person's attributes one by one.
-After entering an attribute, the system will prompt the user input for another attribute.
+In this mode, instead of typing a long command, you can add a person's attributes one by one.
+After entering an attribute, the system will prompt you to input another attribute.
 
 Format: `add t/**TYPE**`
 
-When the user type in the command above, a form will be displayed. For example, if a user wants to add a patient,
-type `add t/patient` to the command box. The form will be as below:
+ORGANice will prompt you to type in each attribute, one by one until all the fields are filled.
+Here are some special commands that might be useful when you are in the process of filling up the form:
 
-image::form.png[width="790"]
+* `/abort` - The command to abandon the current form that you are currently filling.
+* `/undo` - The command to undo the last entry that you entered, this can be useful as sometimes you might make some careless mistakes.
+* `/done` - The command to indicate that you are done with filling all the required attributes and want to add the person into the application.
 
-ORGANice will prompt the user to type in each attribute, one by one until all the fields are filled.
+Note that both `/abort` and `/undo` commands can be used at any point in times, while `/done` command can only be used once all the required attributes have been specified.
 
-For example, when a user wants to add a patient, the display box will prompt the user to enter the patient's name.
-For the user to add a patient that is named "John Doe", type `John Doe` and then press enter.
-After entering the name, the patient's name will be displayed in the form, as shown below.
+That is all the important commands for this feature, now let's see an example on how the feature will look like.
+When you want to add a `Patient`, type `add t/patient` on the `Command Box` and a form will appear as below and the `Result Box` will prompt the user to enter the patient's name.
 
-image::nameField.png[width="790"]
+image::addForm.png[width="790"]
 
-Inputs for each attribute will be checked, if it is a valid input then the user input will automatically reflected
-on the form, if it is not valid, ORGANice will show a warning to the user.
+If you want to add a patient named "John Doe", type `John Doe` and then press enter.
+After entering the name, the patient's name will be displayed on the form as seen below and the progress bar will increase indicating the your progress in filling up the form.
 
-The following screenshot shows the warning that ORGANice will give to the user if the input is not valid:
+image::nameFilled.png[width="790"]
 
-image::wrongInput.png[width="790"]
+Input for each attribute will be checked, if it is a valid input then the user input will automatically be reflected
```

Comment:
Suggestion for this sentence:
Each time we type an input, ORGANice will check if it is a valid input. If the input is valid, it will be reflected on the form. If not, ORGANice will display a warning message stating the criteria for a valid input of that attribute. 

(put an example of a correct input for nric, then explain)

Say you typed in "G999999999A" and pressed kbd:[Enter] when ORGANice prompts you to type in the NRIC of the patient. Three changes occur to ORGANice, which can be seen in the screenshot below:

* Your input will remain in the Command Box, but the colour of the input text turns to red. 
* An error message is displayed in the Message Display Box. This error message details the criteria for a valid Nric input. This error message is specific to the attribute type. For example if you typed in an invalid name, an error message detailing the criteria for a valid name will be displayed. 
* Your input will not be reflected in the form.